contracts/utils/ReentrancyGuard.sol

```diff
@@ -17,7 +17,10 @@ pragma solidity ^0.6.0;
  * https://blog.openzeppelin.com/reentrancy-after-istanbul/[Reentrancy After Istanbul].
  */
 contract ReentrancyGuard {
-    bool private _notEntered;
+    uint256 private constant _NOT_ENTERED = 1;
+    uint256 private constant _ENTERED = uint256(int256(-1));
+    
+    uint256 private _status;
```

Comment:
This is not necessary but will improve the cost of the first execution, and is more correct in my opinion.

```suggestion
    uint256 private _status = _NOT_ENTERED;
```

Reply:
This is unnecessary: the constructor explicitly sets this value.

Reply:
Right. :facepalm: Is there a reason for doing it in the constructor rather than the state variable declaration?

Reply:
I would just do it in whichever is the standard for these libraries. I don't think there is a difference in this case

Reply:
I am personally not a fan of initialization of values outside of constructors in Solidity. Also in the context of upgradeable contracts I like the presence of an explicit constructor as an indicator for the need of an initialization function.